### PR TITLE
Port api modules 2

### PR DESCRIPTION
**Pull request by camden11**
## Description and Context
This switches deps for the following api modules to use `local-dev-lib`
* `api/customObject`
* `api/schema` (consolidated with `customObject`)
* `api/fileTransport`

## Testing
The following commands need to be tested
* `hs custom-object create`
* `hs custom-object scheme create`
* `hs custom-object schema delete`
* `hs custom-object schema fetch`
* `hs custom-object scheme update`

## TODO
Need to merge this PR and release LDL https://github.com/HubSpot/hubspot-local-dev-lib/pull/99

## Who to Notify
@kemmerle @brandenrodgers

### CODE REVIEW

File: packages/cli/commands/customObject/schema/delete.js [hunk 1]
```diff
@@ -25,7 +27,7 @@ exports.handler = async options => {
   trackCommandUsage('custom-object-schema-delete', null, accountId);
 
   try {
-    await deleteSchema(accountId, name);
+    await deleteObjectSchema(accountId, name);
```
Comment (brandenrodgers) [inline packages/cli/commands/customObject/schema/delete.js:30]: Should we swap out `logErrorInstance` for `logApiErrorInstance` here too?

File: packages/cli/commands/customObject/schema/update.js [hunk 2]
```diff
@@ -61,7 +62,8 @@ exports.handler = async options => {
       );
     }
   } catch (e) {
-    logErrorInstance(e, { accountId });
+    console.log(e);
```
Comment (brandenrodgers) [inline packages/cli/commands/customObject/schema/update.js:65]: Looks like a leftover console.log

**Comment by camden11** [comment]
Fixed those things + added the new LDL dep so this should be good to go

**Review by brandenrodgers — APPROVED** [review]
looks great!

**Pull request merged**